cmake/modules/FindTIRPC.cmake

```diff
@@ -0,0 +1,46 @@
+# FindTIRPC
```

Comment:
Should this not be in the sigar package?

Reply:
I was unsure. I put it into the folder with the other FindX.cmake files. Shall I move it to `ext/sigar/`?